src/explore-education-statistics-frontend/src/modules/methodologies/MethodologyIndexPage.tsx

```diff
@@ -71,7 +71,11 @@ const MethodologyIndexPage: NextPage<Props> = ({ themes = [] }) => {
               >
                 {topics.map(
                   ({ id: topicId, title: topicTitle, publications }) => (
-                    <Details key={topicId} id={topicId} summary={topicTitle}>
+                    <Details
+                      key={topicId}
+                      id={`methodology-${topicId}`}
```

Comment:
I think this id tag would be better as `topic-${topicId}`?

Reply:
Don't think this was even being used. I've removed this ID